commons/src/main/java/com/powsybl/commons/datastore/Bzip2FileDataStore.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.datastore;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream;
+import org.apache.commons.compress.compressors.bzip2.BZip2CompressorOutputStream;
+
+import com.powsybl.commons.datasource.Bzip2FileDataSource;
+import com.powsybl.commons.datasource.DataSource;
+import com.powsybl.commons.datasource.DataSourceUtil;
+
+/**
+ * @author Giovanni Ferrari <giovanni.ferrari at techrain.eu>
+ */
+public class Bzip2FileDataStore implements DataStore {
+
+    private final Path path;
+
+    public Bzip2FileDataStore(Path path) {
+        this.path = Objects.requireNonNull(path);
+    }
+
+    @Override
+    public List<String> getEntryNames() throws IOException {
+        return Collections.singletonList(getEntryName());
+    }
+
+    @Override
+    public boolean exists(String entryName) {
+        return getEntryName().equals(entryName);
+    }
+
+    @Override
+    public InputStream newInputStream(String entryName) throws IOException {
+        return new BZip2CompressorInputStream(Files.newInputStream(path));
```

Comment:
As in gzip implementation, I think we should throw an IOException if the entryName does not match the registered one.

Reply:
done